server/rooms.ts

```diff
@@ -824,6 +829,29 @@ export abstract class BasicRoom {
 			}
 		}
 	}
+	sanitizeSection(section: string) {
```

Comment:
Is there a reason this isn't `validateSection`?

Reply:
("sanitize" means something different from what you're doing here)

Reply:
again not really